framework/docker/upgrade_version_test.go

```diff
@@ -24,7 +24,7 @@ func TestUpgradeVersion(t *testing.T) {
 
 	require.NoError(t, wait.ForBlocks(testCfg.Ctx, 5, chain))
 
-	err = chain.Stop(testCfg.Ctx)
+	err = chain.Remove(testCfg.Ctx)
```

Comment:
changed all occurrences of Stop -> Remove to keep same functionality. This will need to be done when we bump the tastora version in celestia-app/celestia-node